### PR TITLE
Update UG to resolve hyperlink bug and also include _ for all the commands

### PR DESCRIPTION
**Pull request by enhao25**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -191,15 +191,15 @@ Format: `exit`
...
 **Tuition Session**
 
 Action | Format, Examples
 --------|------------------
-**List** | `list session`
-**Find** | `find_student KEYWORD`<br><br>e.g. `find session John`
-**Add** | `add session n/STUDENT_NAME d/DATE t/TIME l/LENGTH_OF_SESSION s/SUBJECT f/FEE`<br><br> e.g., `add session n/John Doe d/14-02-2021 t/1800 l/2 s/Biology f/40`
-**Delete** | `delete session INDEX`<br><br>e.g. `delete session 3`
+**List** | `list_session`
+**Find** | `find_student KEYWORD`<br><br>e.g. `find_session John`
```
Comment (samleewy) [inline docs/UserGuide.md]: minor bug: `find_session KEYWORD` instead of `find_student KEYWORD`

**Review by samleewy — CHANGES_REQUESTED** [review]
minor bug in `find_session` as stated, overall looks good to go! 👍

File: docs/UserGuide.md (hunk 1 above)
Comment (enhao25) [reply docs/UserGuide.md]: Waaaa nice catch man! Updated it accordingly already!

**Review by chwoozy — APPROVED** [review]
LGTM! 😄

**Review by samleewy — APPROVED** [review]
LGTM, merge! 🎉

**Pull request merged**